server/datastores/lead.go

```diff
@@ -4,7 +4,7 @@ import "github.com/jmoiron/sqlx"
 
 const createLeadStm = `
 	INSERT INTO leads(id, mail, hashCode, invited)
-	VALUES(nextval('seq_leads'), $1, md5(to_char(now(), 'YYYYMMDDHHMISSMS')), $2)
+	VALUES(nextval('seq_leads'), $1, reverse(lpad((select last_value::varchar from seq_leads),4,'0')), $2)
```

Comment:
this can be easy to guess, isn't it?

maybe we can just generate a random 4 digit string and store it too?